features/facebookImport/src/context/importer-context.jsx

```diff
@@ -134,7 +134,7 @@ export const ImporterProvider = ({ children }) => {
         if (changedState) {
             Object.keys(changedState)?.forEach((key) => {
                 if (!navigationStates.includes(key)) {
-                    console.log(`NavigationStateError with key: ${key}`);
+                    //                  console.log(`NavigationStateError with key: ${key}`);
```

Comment:
Wait, this looks like an actual error to me (should probably use `console.error`) -> But I dunno why. I guess _blame_ would help. If it does show up in practice, sounds like it needs fixing.

Reply:
Also, probably just deleting it would be better than commenting out, right?